examples/siro_sandbox/sandbox_app.py

```diff
@@ -623,56 +639,65 @@ def get_grasp_release_controls_text():
         controls_str += "ESC: exit\n"
         controls_str += "M: next episode\n"
 
-        if self._first_person_mode:
-            # controls_str += "Left-click: toggle cursor\n"  # make this "unofficial" for now
-            controls_str += "I, K: look up, down\n"
-            controls_str += "A, D: turn\n"
-            controls_str += "W, S: walk\n"
-            controls_str += get_grasp_release_controls_text()
-        # third-person mode
-        elif not self.is_free_camera_mode():
-            controls_str += "R + drag: rotate camera\n"
-            controls_str += "Right-click: walk\n"
-            controls_str += "A, D: turn\n"
-            controls_str += "W, S: walk\n"
-            controls_str += "Scroll: zoom\n"
-            controls_str += get_grasp_release_controls_text()
-        else:
-            controls_str += "Left-click + drag: rotate camera\n"
-            controls_str += "A, D: turn camera\n"
-            controls_str += "W, S: pan camera\n"
-            controls_str += "O, P: raise or lower camera\n"
-            controls_str += "Scroll: zoom\n"
+        if not (self.env.episode_over or self.task_complete):
```

Comment:
I don't understand the difference between `episode_over` and `task_complete`. Are you getting at whether the task was successful versus failed?

Reply:
Yes, episode is over when any of the constraints is violated and task is complete when the `composite_success` measure is True.